src/views/Dashboard/Doctor/Account/Verification.vue

```diff
@@ -45,6 +47,12 @@ export default {
   name: 'LabAccount',
   mounted() {
   },
+  data: () => ({
```

Comment:
Please move the rules to `computed`, you can refer to this: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

Reply:
done, all rules move to computed